senza/cli.py

```diff
@@ -510,12 +513,13 @@ def get_classic_load_balancer_metrics(cloudwatch, lb_name, start, now):
 @cli.command()
 @region_option
 @output_option
+@click.option('--field', '-f', metavar='NAME', multiple=True, help='Specify field to be returned')
```

Comment:
you can define the option once at the top of the file and reuse it (like `@region_option`)